cql2pgjson/src/test/resources/compoundIndex.sql

```diff
@@ -41,7 +56,22 @@ BEGIN
   RETURN NEW;
 END; $$ language 'plpgsql';
 CREATE TRIGGER update_tableb_references BEFORE INSERT OR UPDATE ON tableb
-  FOR EACH ROW EXECUTE PROCEDURE update_tableb_references();    
+  FOR EACH ROW EXECUTE PROCEDURE update_tableb_references();   
+ CREATE OR REPLACE FUNCTION update_tablec_references() RETURNS TRIGGER AS $$
+BEGIN
+  NEW.id       = NEW.jsonb->>'id';
+  RETURN NEW;
+END; $$ language 'plpgsql';
+CREATE TRIGGER update_tablec_references BEFORE INSERT OR UPDATE ON tablec
+  FOR EACH ROW EXECUTE PROCEDURE update_tablec_references();
+
+CREATE OR REPLACE FUNCTION update_tabled_references() RETURNS TRIGGER AS $$
+BEGIN
+  NEW.id       = NEW.jsonb->>'id';
+  RETURN NEW;
+END; $$ language 'plpgsql';
+CREATE TRIGGER update_tabled_references BEFORE INSERT OR UPDATE ON tabled
+  FOR EACH ROW EXECUTE PROCEDURE update_tabled_references();    
```

Comment:
Can the four functions and four triggers avoid duplicated code?

For example

    CREATE OR REPLACE FUNCTION update_id() RETURNS TRIGGER AS $$
    BEGIN
      NEW.id = NEW.jsonb->>'id';
      RETURN NEW;
    END; $$ language 'plpgsql';
    CREATE TRIGGER update_tablea_id BEFORE INSERT OR UPDATE ON tablea FOR EACH ROW EXECUTE PROCEDURE update_id();
    CREATE TRIGGER update_tableb_id BEFORE INSERT OR UPDATE ON tableb FOR EACH ROW EXECUTE PROCEDURE update_id();
    CREATE TRIGGER update_tablec_id BEFORE INSERT OR UPDATE ON tablec FOR EACH ROW EXECUTE PROCEDURE update_id();
    CREATE TRIGGER update_tabled_id BEFORE INSERT OR UPDATE ON tabled FOR EACH ROW EXECUTE PROCEDURE update_id();